projects/plugins/protect/src/js/components/pricing-table/index.jsx

```diff
@@ -128,8 +136,8 @@ const ConnectedPricingTable = ( { onScanAdd, scanJustAdded } ) => {
 							fullWidth
 							variant="secondary"
 							onClick={ getProtectFree }
-							isLoading={ protectFreeIsRegistering }
-							disabled={ protectFreeIsRegistering || scanJustAdded }
+							isLoading={ getProtectFreeButtonIsLoading }
```

Comment:
Nice, these new state variables make things a lot easier to read/understand! :+1: